var/spack/repos/builtin/packages/cbtf-argonavis-gui/package.py

```diff
@@ -49,20 +49,55 @@ class CbtfArgonavisGui(QMakePackage):
     """
     homepage = "http://sourceforge.net/p/cbtf/wiki/Home/"
 
+    version('1.3.0.0', branch='1.3.0.0',
+            git='https://github.com/OpenSpeedShop/cbtf-argonavis-gui.git')
+
     version('1.3.0', branch='master',
             git='https://github.com/OpenSpeedShop/cbtf-argonavis-gui.git')
 
-    depends_on("cmake@3.0.2:", type='build')
-    depends_on("openspeedshop+cuda gui='qt4'")
-    depends_on('qt@4.8.6:')
-    depends_on("boost@1.50.0:")
-    depends_on("cbtf")
-    depends_on("cbtf-krell")
-    depends_on("cbtf-argonavis")
+    version('develop', branch='master',
+            git='https://github.com/OpenSpeedShop/cbtf-argonavis-gui.git')
+
+    depends_on("cmake@3.0.2:", when='@develop', type='build')
+    depends_on("cmake@3.11.1", when='@1.3.0.0', type='build')
+
+    # To specify ^elfutils@0.170 on the command line spack 
+    # apparently needs/wants this dependency explicity here
+    # even though it is referenced downstream
+    depends_on("elf", type="link")
```

Comment:
Where exactly downstream? As mentioned [here](http://spack.readthedocs.io/en/latest/known_issues.html#default-variants-are-not-taken-into-account-during-concretization), there are known issues with this. Even then, it isn't a good idea to hack the package to work around them.